doc/ideas/gameplay.md

```diff
@@ -152,16 +208,80 @@ Multiplayer
+
+ - depending on the architecture of the multiplayer it could be even possible to spare out all that client modeling and just integrate an basic in-game browser which connects just to an OpenAge-community-server and from there you could manage everything. To play a game the user could just click on a link from the game browser which gives data to an internal protocol like openage://<game-id>/<foo>/<bar>/<link-to-settings-file>/<...>
+
+     - this would result in a thinner client
+     - but: taking too much multiplayer features out of the client leaves it helpless for private matches -> thin grade
+
+
+### Spectating/Casting
+
+ - the casting mode could be an advanced spectating mode, with more features than just replaying/live-spectating the game
+   - Spectator layer: For the general public watching the game. This could be a mirror of the casting layer that the casters see. You can either choose to lock the viewpoint to one of the casters' cameras or move the camera freely around the map. Also you could have the ability to chat with other spectators and draw notes on the map, mark your own points of interest. Sync is not as important here. (_ColonelPanic_ @reddit)
+
+
+ - ZOOM! (also in-game not just for casters)
+ - maybe even turning the game in 90° angles
+   - the old models could stay 2D and just be fit into the turned environment(?)
+
+ - the feature set of CaptureAge could be understood as a spectator mode
+   - if they add Picture-in-Picture and replay-functions goes more into direction what's understood in this context as casting mode
+
+ - casting mode should help you to either work on your own or collaboratively work together with one or more Co-Caster(s) on the presentation of the actions in a game
+
+   - Casting layer: For casters and co casters. Could support features like pinging something on the map for the co caster, marking points of interest/units/buildings, show picture-in-picture views of the other casters and so on. An important requirement would be that the casters' views are synced. (_ColonelPanic_ @reddit)
+   - casters should have the possibility to join a live-game together as a caster-team
+   - e.g. two separate OpenAge-accounts could be temporarily bound together (master-slave-principle)
+     - they both work together at the master-device (streaming device from main caster), each caster stil has it's own independant view of the game
+
+   - there should be an option to cut scenes together in a live game, so that if one caster sees an action somewhere on the map, he/she gives a command to the game that itself clips (saves camera position and game status) it and puts it into a queue on the master-device to replay it
+
+   - outside of the main spectating window could be a second window (used for two monitor solutions) with the waiting list for the replays and all the other options and features special to the casting mode
+
+ - in-game lag-free voice-communication between casters
+
```

Comment:
One additional idea I would add is the collection of more player-specific information like

* actions per minute (APM)
* current camera location
* small indicators for units that show if they are visible to another player, even if the FOW is turned off